Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/GDSInstructions/GDSInstructionsViewModel.swift

```diff
@@ -17,7 +17,7 @@ import UIKit
 /// for other code such as making an API call.
 /// If `rightBarButtonTitle` is `nil` then the navigation item right bar button is hidden
 /// Otherwise, it is set to the title property and the action is set to the `didDismiss()` action.
-public protocol GDSInstructionsViewModel {
+public protocol GDSInstructionsViewModel: BaseViewModel {
```

Comment:
I don't think we want to have all these view models automatically conform to this. I think it should be up to the developer to add it if they need it.

It's [interface segregation principle](https://en.wikipedia.org/wiki/Interface_segregation_principle). The point of having both protocols is that they're both optional.